Python/Tests/PythonToolsMockTests/NavigableTests.cs

```diff
@@ -27,44 +26,26 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Classification;
-using Microsoft.VisualStudio.Text.Editor;
-using Microsoft.VisualStudioTools.MockVsTests;
 using TestUtilities;
 using TestUtilities.Python;
 
 namespace PythonToolsMockTests {
     [TestClass]
     public class NavigableTests {
-        private MockVs _vs;
-
         [ClassInitialize]
         public static void DoDeployment(TestContext context) {
             AssertListener.Initialize();
             PythonTestData.Deploy(includeTestData: false);
         }
 
-        [TestInitialize]
-        public void TestInit() {
-            MockPythonToolsPackage.SuppressTaskProvider = true;
-            VsProjectAnalyzer.SuppressTaskProvider = true;
-            _vs = new MockVs();
-        }
-
-        [TestCleanup]
-        public void TestCleanup() {
-            MockPythonToolsPackage.SuppressTaskProvider = false;
-            VsProjectAnalyzer.SuppressTaskProvider = false;
-            _vs.Dispose();
-        }
-
-        [Ignore] // This works in the IDE, need to figure out why it doesn't here
+        [Ignore] // TODO: Figure out why we don't get any definition, this works in IDE
```

Comment:
I just tried it, and it did not help. I suppose you need a real interpreter in order to find standard library modules? Right now PythonEditor uses `InterpreterFactoryCreator.CreateAnalysisInterpreterFactory` because I'm not passing a factory.